website.js

```diff
@@ -395,6 +410,18 @@ app.get('/admin/modules', function(req, res) {
     });
 });
 
+app.get('/admin/poll', function(req, res) {
+    connection.query('select * from poll', function(err, result) {
+        res.render('admin/poll/index.html')
+    });
+});
+
+app.get('/admin/poll/create', function(req, res) {
```

Comment:
You could follow the Rails URL conventions like this, but it's not required. Just a heads up:

| Path       | Method | used for              |
----------|---------|-----------------|
| `/polls` | GET       | displays all polls |
| `/polls/new` | GET       | shows the form to create a new poll |
| `/polls` | POST       | creates a new poll |
| `/polls/:id` | GET       | shows one poll |
| `/polls/:id/edit` | GET       | shows the form to edit a new poll |
| `/polls/:id` | PUT       | updates a specific poll |
| `/polls/:id` | DELETE       | deletes a specific poll |